.github/workflows/test.yml

```diff
@@ -0,0 +1,20 @@
+name: Test
+
+on: pull_request
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        version: [ 8, 11, 17 ]
+        os: [ ubuntu-latest, windows-latest ]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
```

Comment:
Distribution is not optional and has to be supplied. Chose `zulu` for no specific reason, can change it to `temurin` or anything else